auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -380,6 +399,10 @@ export default class EntityManagement extends Mixins(AccountChangeMixin, NextPag
   .view-header {
     justify-content: space-between;
 
+    #add-name-request-btn {
+      font-weight: normal;
```

Comment:
There's a Vuetify class for this if you don't want this class her.

Reply:
oh nice, i see it. Font weight regular? 
I'd prefer the helper class. Updated!